src/sidebar/store/test/use-store-test.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+const { mount } = require('enzyme');
+const { createStore } = require('redux');
+const { createElement } = require('preact');
+const { act } = require('preact/test-utils');
+
+const useStore = require('../use-store');
+const { $imports } = useStore;
```

Comment:
Some day you'll need to explain this magic to me, but I like what it does!

Reply:
`$imports` gets added by a Babel plugin ("babel-plugin-mockable-imports") when running tests. It is the frontend version of `patch`. The plugin's [README](https://github.com/robertknight/babel-plugin-mockable-imports/blob/master/README.md) explains the motivation, usage and implementation (and also why we're not using Proxyquire any more).

For any Babel plugin, you can see what it does to code by running `babel` locally. For example:

```
npm install -g @babel/cli
babel --no-babelrc --plugins mockable-imports src/sidebar/util/is-third-party-service.js
```